src/App.tsx

```diff
@@ -461,232 +286,253 @@ export default function App() {
+            </Route>
+            <Route exact path={APP_PATHS.NURSERY_WITHDRAWALS_DETAILS}>
+              <NurseryWithdrawalsDetails species={species} plotNames={plotNames} />
+            </Route>
+            <Route exact path={APP_PATHS.NURSERY_REASSIGNMENT}>
+              <NurseryReassignment />
+            </Route>
+            <Route exact path={APP_PATHS.CONTACT_US}>
+              <ContactUs />
+            </Route>
+            <Route exact path={APP_PATHS.MY_ACCOUNT_EDIT}>
+              <MyAccount organizations={organizations} edit={true} reloadData={reloadData} />
+            </Route>
+            <Route exact path={APP_PATHS.MY_ACCOUNT}>
+              <MyAccount organizations={organizations} edit={false} />
+            </Route>
+
+            {!isProduction && (
+              <Route exact path={APP_PATHS.OPT_IN}>
+                <OptInFeatures refresh={reloadPreferences} />
+              </Route>
+            )}
+
+            {/* Redirects. Invalid paths will redirect to the closest valid path. */}
+            {/* In alphabetical order for easy reference with APP_PATHS, except for /home which must go last */}
+            <Route path={APP_PATHS.ACCESSIONS + '/'}>
+              <Redirect to={APP_PATHS.ACCESSIONS} />
+            </Route>
+            <Route path={APP_PATHS.CHECKIN + '/'}>
+              <Redirect to={APP_PATHS.CHECKIN} />
+            </Route>
+            <Route path={APP_PATHS.CONTACT_US + '/'}>
+              <Redirect to={APP_PATHS.CONTACT_US} />
+            </Route>
+            <Route exact path={APP_PATHS.ORGANIZATION + '/'}>
+              <Redirect to={APP_PATHS.ORGANIZATION} />
+            </Route>
+            <Route exact path={APP_PATHS.PEOPLE + '/'}>
+              <Redirect to={APP_PATHS.PEOPLE} />
+            </Route>
+            <Route path={APP_PATHS.SEEDS_DASHBOARD + '/'}>
+              <Redirect to={APP_PATHS.SEEDS_DASHBOARD} />
+            </Route>
+            <Route path={APP_PATHS.SPECIES + '/'}>
+              <Redirect to={APP_PATHS.SPECIES} />
+            </Route>
+            <Route path='/'>
+              <Redirect to={APP_PATHS.HOME} />
+            </Route>
+          </Switch>
+        </ErrorBoundary>
       </div>
     </>
   );
 
   return (
     <StyledEngineProvider injectFirst>
       <CssBaseline />
-      <UserProvider data={{ user, reloadUser }}>
-        {selectedOrganization && (
-          <OrganizationProvider data={{ selectedOrganization, setSelectedOrganization, organizations, reloadData }}>
-            <LocalizationProvider locale={locale}>{getContent()}</LocalizationProvider>
-          </OrganizationProvider>
-        )}
-      </UserProvider>
+      <ToastSnackbar />
+      <TopBar>
+        <TopBarContent setShowNavBar={setShowNavBar} />
+      </TopBar>
+      <div className={classes.container}>{organizations.length === 0 ? getOrphanedUserContent() : getContent()}</div>
     </StyledEngineProvider>
   );
 }
```

Comment:
cleaned up rendering of different paths in one place instead of scattered across the app